frontend/src/components/CommentsOnPost.jsx

```diff
@@ -44,6 +49,7 @@ class CommentsOnPost extends Component {
 							comments: commentList,
 							isFetching: false,
 						});
+						this.updateComments(this.state.comments);
```

Comment:
why not this.props.updateComments(newComments); directly lol

Reply:
Just in case I want updateComments to do other things

Reply:
sure